Automattic/jetpack
# Fix/share this available on password protected posts

Fixes [wp-calypso-#56386](https://github.com/Automattic/wp-calypso/issues/56386)

#### Changes proposed in this Pull Request:
Adds a check for a password before showing sharing buttons.



#### Testing instructions:
*Go to http://localhost/wp-admin/admin.php?page=jetpack#/sharing and toggle on "Add sharing buttons to your posts and pages"
* Create a post and password protect it.
*View the post in an incognito window, and do not enter the password.
*You should no longer be able to see the sharing buttons.
Before:
![image](https://user-images.githubusercontent.com/72156679/135556392-fbcfe813-91bb-4fab-993b-62744434b07c.png)


After:
![image](https://user-images.githubusercontent.com/72156679/135556315-f644d371-0a42-478d-a6b4-08c8dbb2e839.png)
